public/locales/cs/translation.json

```diff
@@ -135,12 +135,15 @@
 		"Application type": "Typ aplikace",
 		"Failed to fetch clients": "Nepodařilo se načíst klienty",
 		"Client name": "Název klienta",
+		"Client ID": "ID klienta",
+		"Client URI": "URI klienta",
 		"New client": "Nový klient",
 		"Clients": "Klienti",
 		"Search": "Vyhledat"
 	},
 	"ClientCreateContainer": {
 		"Something went wrong, failed to fetch clients": "Něco je špatně, nepodařilo se získat klienty",
```

Comment:
@byewokko This can be removed, am I correct? Since you have removed / redone the translation for english version